x-pack/plugins/upgrade_assistant/public/application/components/overview/backup_step/backup_step.tsx

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useState, useEffect } from 'react';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { i18n } from '@kbn/i18n';
+import { EuiText, EuiButton, EuiSpacer } from '@elastic/eui';
+import type { EuiStepProps } from '@elastic/eui/src/components/steps/step';
+
+import { useAppContext } from '../../../app_context';
+
+const i18nTexts = {
+  backupStepTitle: i18n.translate('xpack.upgradeAssistant.overview.backupStepTitle', {
+    defaultMessage: 'Back up your data',
+  }),
+
+  backupStepDescription: i18n.translate('xpack.upgradeAssistant.overview.backupStepDescription', {
+    defaultMessage: 'Back up your data before addressing any deprecation issues.',
```

Comment:
I'll defer to @debadair, but I think in some places we've changed the copy to "deprecation warnings" rather than "deprecation issues".

Reply:
Makes sense. I did a quick search of the codebase and found numerous references to "deprecation issues" outside the scope of this PR. How about I address them all in their own PR?

Reply:
👍 sounds good, thanks!